pkg/bbgo/interact.go

```diff
@@ -130,11 +113,11 @@ func (it *CoreInteraction) Commands(i *interact.Interact) {
 	i.PrivateCommand("/position", "Show Position", func(reply interact.Reply) error {
 		// it.trader.exchangeStrategies
 		// send symbol options
-		if strategies, found := it.FilterStrategyByInterface((*PositionReader)(nil)); found {
+		if strategies, found := it.FilterStrategyByCallBack("GetPositionCallback"); found {
```

Comment:
I think the FilterStrategyByInterface method already make sense.

Reply:
but why do we need those interfaces if we want to move to callback style?

Reply:
we need the interface to filter the strategies, so I think the original implementation is good. it does the interface checking and filtering.

Reply:
will change it back